doc/api.apib

```diff
@@ -20,7 +20,7 @@ aglio -i doc/api.apib -o api.html --theme-template triple --theme-variables stre
 
 # Group Workflows
 
-## Workflow [/v1/workflows/{component}/{workflow_id}]
```

Comment:
I think it's probably OK for now to have v2 only, and an issue to track this. #118